docs/content/docs/get-started/installation.md

```diff
@@ -22,47 +22,46 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Installation
-
+## Overview
 Flink Agents provides both Python and Java APIs to define a Flink Agents job.
 
-To try Flink Agents in Python on a local executor, you only need to install the Flink Agents Python package.
-
-To define a Flink Agents job using the Java API, or to run the job in a Flink cluster, you need to install both the Flink Agents Python and Java dependencies. 
+To try Flink-Agents in Python or Java, user can only install the correspond package.
```

Comment:
This is incorrect. To run a Python job in a Flink cluster, you'll also need the java packages.

I think it's too complicated to explain when to install python / java only, and when to install both. The benefit for users to install one less package is probably weaker than the confusion for users to understand what packages they need. So let's just tell our users to always install both.